README.md

```diff
@@ -1,2 +1,46 @@
 # Women-summit
 women's summit to highlight some of the problems facing women
+
```

Comment:
- [ ] [OPTIONAL] Good job 👏, all requirements from the previous review are implemented ✔. I would suggest adding a screenshot for the website. Let's also add Twitter and LinkedIn account links in the **author** part 🙌